lib/puppet/provider/service/init.rb

```diff
@@ -61,6 +61,11 @@ def self.excludes
     excludes += %w{cryptdisks-udev}
     excludes += %w{statd-mounting}
     excludes += %w{gssd-mounting}
+    # Prevent puppet failing on unsafe scripts from Yocto Linux
+    excludes += %w{banner.sh bootmisc.sh checkroot.sh devpts.sh dmesg.sh
+                 hostname.sh mountall.sh mountnfs.sh populate-volatile.sh
+                 rmnologin.sh save-rtc.sh sendsigs sysfs.sh umountfs
+                 umountnfs.sh}
```

Comment:
FWIW, you don't see *.sh files in init normally, so most of these would be just WRL specific. However, for safety it does seem like we should confine this to a specific platform.